experiments/AMIP/moist_mpi_earth/coupler_utils/conservation_checker.jl

```diff
@@ -0,0 +1,156 @@
+# generalise this into coupler-specific function
+
+abstract type AbstractCheck end
+
+struct OnlineConservationCheck{A} <: AbstractCheck
+    ρe_tot_atmos::A
+    ρe_tot_land::A
+    ρe_tot_ocean::A
+    ρe_tot_seaice::A
+    toa_net_source::A
+    ice_base_source::A
+end
+function check_conservation(
+    cs::OnlineConservationCheck,
+    coupler_sim,
+    atmos_sim = nothing,
+    land_sim = nothing,
+    ocean_sim = nothing,
+    seaice_sim = nothing,
+    radiation = true,
+)
+    z = parent(Fields.coordinate_field(face_space).z)
```

Comment:
`face_space` is not defined here - it is probably passed as a global currently.

Reply:
yes...we'll have to revisit a few of those! FT is also a global, and mask.